client/angular/controllers/AdminController.js

```diff
@@ -22,6 +22,29 @@
         $scope.errorMessage = "";
 
         $scope.deleteTweet = adminDashDataService.deleteTweet;
+        $scope.blockedUsers = [];
+
+        $scope.getBlockedUsers = function() {
+            adminDashDataService.blockedUsers().then(function (users) {
+                $scope.blockedUsers = users;
+            });
+        };
+
+        $scope.removeBlockedUser = function(user) {
+            adminDashDataService.removeBlockedUser(user).then(function (result) {
+                adminDashDataService.blockedUsers().then(function (users) {
+                    $scope.blockedUsers = users;
+                });
+            });
+        };
+
+        $scope.addBlockedUser = function(name, screen_name) {
```

Comment:
Is there a reason to pass in the name with the screen name? When it gets to the tweetSearcher it only uses the screen_name.